README.md

```diff
@@ -1377,3 +1377,6 @@ I am sure it will be an awesome journey!
 # Hello there!
 1st time working via command line, [Vasiliy](https://github.com/Vas-P) =)
 here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-printing.jpg)
+
+# Hi guys!
+nice to join such a wonderfull jorney! ![Cat] (https://www.google.com/url?sa=i&url=https%3A%2F%2Fgiphy.com%2Fgifs%2Fcomputer-cat-wearing-glasses-VbnUQpnihPSIgIXuZv&psig=AOvVaw1qjuR2HA8jkhVMxiUD9ZiX&ust=1602664350071000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCOiu1o2UsewCFQAAAAAdAAAAABAD)
```

Comment:
Hi, there is a url with a lot of parameters which redirects user to `giphy.com`, thats why it couldn't be displayed as a markdown preview image. Please find the source path of an image and use it

Reply:
fixed, thank you!